src/regression_tracker.py

```diff
@@ -31,6 +31,25 @@ def _stop(self, sub_id):
         if sub_id:
             self._api_helper.unsubscribe_filters(sub_id)
 
+    def _collect_errors(self, node):
+        """Returns a dict containing the 'error_code' and 'error_msg'
+        data fields of <node>. If <node> doesn't have any info in them,
+        it searches upwards through parent nodes until it reaches a node
+        that has them.
+        """
+        if node['data'].get('error_code'):
+            return {
+                'error_node': node['data']['error_code'],
+                'error_msg': node['data']['error_msg']
+            }
+        elif node.get('parent'):
+            parent = self._api.node.get(node['parent'])
+            return self._collect_errors(parent)
```

Comment:
I am not sure if we should set error information of parent node.
Specifically for `kbuild` nodes, for example, the parent node of `https://staging.kernelci.org:9000/latest/node/65e6d3578c896c56c9f5e9ff` has completely different runtime than the child one.

Reply:
Okay. Let's keep it like this as of now.